File: output/gemma4-e2b/avatar/Python/Java/atcoder_ABC164_A.java
import java.util.Scanner;

public class atcoder_ABC164_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Read S and W from standard input
        if (scanner.hasNextInt()) {
            int S = scanner.nextInt();
            if (scanner.hasNextInt()) {
                int W = scanner.nextInt();
                
                // Logic: print "unsafe" if S <= W else print "safe"
                if (S <= W) {
                    System.out.println("unsafe");
                } else {
                    System.out.println("safe");
                }
            }
        }
        
        scanner.close();
    }
}

File: output/gemma4-e2b/avatar/Java/Python/atcoder_ABC164_A.py
import sys

try:
    # Read all input lines
    input_lines = sys.stdin.read().splitlines()
    
    if input_lines:
        # Process the first line
        line = input_lines[0].strip()
        if line:
            numlist = line.split()
            if len(numlist) >= 2:
                Sheep = int(numlist[0])
                Wolve = int(numlist[1])
                
                if Wolve >= Sheep:
                    print("unsafe")
                else:
                    print("safe")
except Exception:
    # Handle potential errors during reading or conversion
    pass